IHP/QueryBuilder.hs

```diff
@@ -268,11 +276,13 @@ toSQL :: forall table. (KnownSymbol table) => QueryBuilder table -> (ByteString,
 toSQL queryBuilder = toSQL' (buildQuery queryBuilder)
 {-# INLINE toSQL #-}
 
-toSQL' sqlQuery@SQLQuery { selectFrom, orderByClause, limitClause, offsetClause } =
+toSQL' sqlQuery@SQLQuery { selectFrom, distinctClause, distinctOnClause, orderByClause, limitClause, offsetClause } =
         (DeepSeq.force theQuery, theParams)
     where
         !theQuery =
-            "SELECT " <> selectors <> " FROM "
+            "SELECT "
+            <> distinctClause' <> distinctOnClause' <> " "
```

Comment:
There is probably a way to merge these two into one, but that is beyond me at the moment.